### PR TITLE
Readme images

### PR DESCRIPTION
**Pull request by blackburn32**
This PR does some touch ups to a slew of the integrations and adds images and writeups to the root level README for all the samples*.

*I have excluded two integrations from the full treatment: 

BPMN converter, which I am working on getting set up still. That one will get some README updates and images in the near future.

in-a-circle which I am going to remove from the repo, and replace with a more comprehensive version of the same extension

### CODE REVIEW

**Review by jselvaraaj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,75 @@
...
+Our Lucid cards for Asana integration is a production ready implementation of a cards integration built to import and manage Asana tasks in Lucidspark.
+The integration uses Lucid's Extension API, and leverages the built in cards framework to allow users to bring Asana data into Lucidspark and to keep that data up to date when changes are made.
+
+![Lucid Cards for Asana](https://lucid.co/static/b20c5a40f333fba7b7799519d8fbd2e0/content.png)
+
+You can find more information about the integration, and install it to use for yourself at its [marketplace listing](https://lucid.co/marketplace/944af3b8/lucid-cards-for-asana).
+
+## BPMN Converter
+
+A tool for importning BPMN formatted files into Lucid using the REST API.
```
Comment (cball18) [inline README.md]:
```suggestion
A tool for importing BPMN formatted files into Lucid using the REST API.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+
+A collection of custom shapes that can be used in Lucidchart.
+These shapes cover data linking, conditional formatting, images, dynamic rendering and more.
+You can find more information about designing your own custom shapes [here](https://developer.lucid.co/custom-shapes/).
+
+![Example shape library](https://cdn-cashy-static-assets.lucidchart.com/open-source-github-repositories/sample-lucid-extensions/example_shape_library.png)
+
+## Google Embed
+
+An integration which brings Google Docs documents into Lucidchart and Lucidspark as expandable links using our Extension API.
```
Comment (cball18) [inline README.md:51]:
```suggestion
An integration which brings Google documents into Lucidchart and Lucidspark as expandable links using our Extension API.
```

File: example-card-integration/README.md [hunk 3]
```diff
@@ -1,6 +1,26 @@
 # Lucid / Example card integration
 
-This repository contains the source for an example card integration built on Lucid's extensibility platform. This integration is based on the card integration walkthrough presented [here](https://developer.lucid.co/extension-api/#lucid-card-integrations). This integration does not connect to an actual external data source and instead mocks the responses one would get from such a source.
+This repository contains the source for an example card integration built on Lucid's extensibility platform.
```
Comment (cball18) [inline example-card-integration/README.md:3]: Should we say Dev platform here to be more broad?

**Review by cball18 — APPROVED** [review]

**Review by SayemW — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (blackburn32) [reply README.md:51]: I can see this one either way. Google Docs is technically the name for the product we're integrating with so I'm hesitant to remove that identifier. I could see removing "documents" though... Unless others feel strongly I think I'll leave this as is for now.

**Pull request merged**